client/src/components/LogFileList.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import WidgetListGroupItem from 'components/WidgetListGroupItem';
+import LogFile from '../api/v2/files';
+
+/**
+ * @typedef {object} LogFileListProps
```

Comment:
The `{boolean?} loading` property for `LogFileList` is not documented here. If we don't want the `loading` argument we could have `files` be set to `null` when waiting for a response.

Reply:
Whoops, forgot to add that! But good point, will just set `files` to null.